### ISSUE TITLE
Asset path wrong

### ISSUE THREAD

**Issue by bd8** [raulrotundo/laravel5starter#51]
In the blade templates you use {{ asset('public/assets...'), I think it must be {{ asset('assets/...').

**Comment by raulrotundo** [comment]
Yes, you're right, this is because a was doing some test and I couldn't change the root path in the heroku app. I will do it my friend, Thanks @bd8 !
